dbms/src/Functions/FunctionsDuration.h

```diff
@@ -88,98 +89,6 @@ class FunctionMyDurationToSec : public IFunction
     void executeImpl(Block & block, const ColumnNumbers & arguments, size_t result) const override;
 };
 
-struct ExtractMyDurationImpl
```

Comment:
Maybe move to a separate file?

Reply:
do you have a nice name for this new file?:) I think it does not fit in the utils directory and does not represent a new class in concept...